CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# PR [#36](https://github.com/theplant/appkit/pull/36)
+
+* Upgrade Airbrake package.
+* Change `airbrakeNotifier.Notify` func to async.
+* `errornotifier.NewAirbrakeNotifier` func now returns `io.Closer` to
+  wait send data to Aribrake.
+* Add a new parameter `context` to `errornotifier.Notifier.Notify` allow
+  add context when call `Notify`.
+* Add log req_id and span_context to `errornotifier.NotifyOnPanic` and
```

Comment:
Are some of these breaking changes? So clients will have to update? If so, please list them as breaking changes here.

Reply:
Added breaking changes in https://github.com/theplant/appkit/pull/36/commits/196540fa4ecb669e767e8e42fbbc8fcc797bfa24